swo_aws_extension/flows/jobs/billing_journal/models.py

```diff
@@ -46,9 +46,10 @@ class SearchItem:
 
 
 @dataclass
-class SearchSubscription:
-    """Billing charge search subscription."""
+class SearchSource:
+    """Billing charge search source."""
 
+    type: Literal["Agreement", "Asset", "Subscription"]
```

Comment:
Why not use a enum?

I know it's a backport...just in case you want to change it next time you're working  in main